app/models/katello/pool.rb

```diff
@@ -50,13 +50,16 @@ class Pool < Katello::Model
 
     validates_lengths_from_database
 
-    validates :subscription, :presence => true
+    validates :subscription, :presence => { :message => N_("cannot be blank. If this is during a manifest import or refresh, try running foreman-rake katello:reimport.") }
```

Comment:
This isn't an error that a user should ever see! As you found, it was (incorrectly) being thrown due to some underlying bad data. Not great, but IMO better that it failed so that you could fix the underlying problem. What I'm getting at is that I don't think this message is needed and katello:reimport wouldn't fix it (right?)